.eslintrc

```diff
@@ -1,37 +1,37 @@
 {
 	"root": true,
-  "extends": "airbnb",
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "parser": "babel-eslint"
+	"extends": "airbnb",
+	"parserOptions": {
+		"ecmaVersion": 6,
+		"parser": "babel-eslint"
 },
 	"env": {
-	    "browser": true,
-	    "node": true,
-	    "mocha": true
-    },
-    "rules": {
-    "one-var": 0,
-    "one-var-declaration-per-line": 0,
-    "new-cap": 0,
-    "consistent-return": 0,
-    "no-param-reassign": 0,
-    "comma-dangle": 0,
-    "curly": ["error", "multi-line"],
-    "import/no-unresolved": [2, { "commonjs": true }],
-    "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
-    "valid-jsdoc": ["error", {
-      "requireReturn": true,
-      "requireReturnType": true,
-      "requireParamDescription": false,
-      "requireReturnDescription": true
-    }],
-    "require-jsdoc": ["error", {
-        "require": {
-            "FunctionDeclaration": true,
-            "MethodDefinition": true,
-            "ClassDeclaration": true
-        }
-    }]
-  }
+		"browser": true,
+		"node": true,
+		"mocha": true
+		},
+		"rules": {
+		"one-var": 0,
+		"one-var-declaration-per-line": 0,
+		"new-cap": 0,
+		"consistent-return": 0,
+		"no-param-reassign": 0,
+		"comma-dangle": 0,
+		"curly": ["error", "multi-line"],
+		"import/no-unresolved": [2, { "commonjs": true }],
+		"no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+		"valid-jsdoc": ["error", {
+			"requireReturn": true,
+			"requireReturnType": true,
+			"requireParamDescription": false,
+			"requireReturnDescription": true
+		}],
+		"require-jsdoc": ["error", {
+			"require": {
+				"FunctionDeclaration": true,
+				"MethodDefinition": true,
+				"ClassDeclaration": true
+			}
+		}]
+	}
```

Comment:
@fridolinho please cross check the eslint setting and confirm if these are right.

Reply:
thanks, I am working on it

Reply:
The indentation errors you can work on it.

Reply:
fixed